components/provider_fs/Cargo.toml

```diff
@@ -61,7 +61,6 @@ criterion = "0.3.3"
 [features]
 export = ["static_assertions", "log"]
 export-bin = ["export", "clap", "icu_provider_cldr", "simple_logger"]
-serialize_none = ["icu_provider_cldr/serialize_none"]
```

Comment:
Suggestion (required-if-valid): Line 25 of this file: I think you can now remove the bincode feature filters, since serialize_none is no longer required.

Reply:
Done.